server/test/lib/scene/scene.duplicate.test.js

```diff
@@ -0,0 +1,24 @@
+const { expect, assert } = require('chai');
+const EventEmitter = require('events');
+const SceneManager = require('../../../lib/scene');
+
+const event = new EventEmitter();
+
+describe('SceneManager', () => {
+  it('should duplicate a scene', async () => {
+    const sceneManager = new SceneManager({}, event);
+
+    const duplicatedScene = await sceneManager.duplicate('to-duplicate-scene', 'new-name', 'new-icon');
+    expect(sceneManager.scenes['new-name']).not.to.be.undefined; // eslint-disable-line
+
+    expect(duplicatedScene).to.have.property('selector', 'new-name');
+    expect(duplicatedScene).to.have.property('name', 'new-name');
+    expect(duplicatedScene).to.have.property('icon', 'new-icon');
```

Comment:
Can you add a bit more checks? What about the triggers? What about the actions?

It should be strictly verified with a .deep.equal() imo :)